terminatorlib/terminal_popup_menu.py

```diff
@@ -13,6 +13,7 @@
 from .config import Config
 from .prefseditor import PrefsEditor
 from . import plugin
+from layoutlauncher import LayoutLauncher
```

Comment:
this neeeds to be from `from .layoutlauncher import LayoutLauncher`

Reply:
Sounds good to me. I don't really know much about how Github works; do I need to do anything?

Reply:
```suggestion
from .layoutlauncher import LayoutLauncher
```